libmscore/score.cpp

```diff
@@ -3972,7 +3972,8 @@ void MasterScore::setPos(POS pos, Fraction tick)
     if (tick < Fraction(0,1)) {
         tick = Fraction(0,1);
     }
-    Q_ASSERT(tick <= lastMeasure()->endTick());
+    Fraction otherTick = lastMeasure()->endTick();
+//    Q_ASSERT(tick <= lastMeasure()->endTick());
```

Comment:
See https://github.com/musescore/MuseScore/pull/6248/commits/bc5bf365ce16f587291d5ebd4cf4e4e85a69e065 for a better fix